### PR TITLE
Feat: Switch logging library to k/klog

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: pkg/cmd/server/start.go [hunk 1]
```diff
@@ -70,11 +71,7 @@ func StartApiServer(etcdPath string, apis []*builders.APIGroupBuilder, openapide
 
 	// To disable providers, manually specify the list provided by getKnownProviders()
 	cmd, _ := NewCommandStartServer(etcdPath, os.Stdout, os.Stderr, apis, wait.NeverStop, title, version)
-	if logflag := flag.CommandLine.Lookup("v"); logflag != nil {
```
Comment (yue9944882) [inline pkg/cmd/server/start.go]: removing explicit logging flags registration here

File: pkg/cmd/server/start.go [hunk 2]
```diff
@@ -132,6 +129,19 @@ func NewCommandStartServer(etcdPath string, out, errOut io.Writer, builders []*b
...
+
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+	flags.AddGoFlagSet(klogFlags)
+
+	// Sync the glog and klog flags.
+	klogFlags.VisitAll(func(f *flag.Flag) {
+		goFlag := flag.CommandLine.Lookup(f.Name)
+		if goFlag != nil {
+			goFlag.Value.Set(f.Value.String())
```
Comment (yue9944882) [inline pkg/cmd/server/start.go]: and klog flags are sync'd to golang flags in order to fix the compatibility issue

**Comment by adohe** [comment]
just add feature issue: [apiserver-builder-alpha#306](https://github.com/kubernetes-incubator/apiserver-builder-alpha/issues/306)

File: pkg/cmd/server/start.go [hunk 3]
```diff
@@ -34,12 +33,14 @@ import (
...
 
-	"github.com/golang/glog"
+	"k8s.io/klog"
 	"github.com/kubernetes-incubator/apiserver-builder-alpha/pkg/validators"
 	"k8s.io/apimachinery/pkg/util/wait"
 	openapinamer "k8s.io/apiserver/pkg/endpoints/openapi"
 	"k8s.io/apiserver/pkg/util/logs"
 	openapi "k8s.io/kube-openapi/pkg/common"
+	_ "k8s.io/apiserver/pkg/util/logs"
+	"flag"
```
Comment (adohe) [inline pkg/cmd/server/start.go]: import order

**Comment by adohe** [comment]
cool~ 👍 overall lgtm. cc @pwittrock for approve

**Pull request merged**